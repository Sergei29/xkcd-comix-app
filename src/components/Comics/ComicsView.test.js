import { shallow } from "enzyme";
import React from "react";

import ComicsView from "./ComicsView.component";

describe("ComicsView", () => {
	let wrapper;
	const mockProps = {
		title: "ABC",
		img: "https:://my-img.png",
		num: 123,
		alt: "BCD",
	};

	beforeEach(() => {
		wrapper = shallow(<ComicsView {...mockProps} />);
	});

	it("should render the component without crashing", () => {
		expect(wrapper.length).toEqual(1);
	});

	it("should match the snapshot", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
