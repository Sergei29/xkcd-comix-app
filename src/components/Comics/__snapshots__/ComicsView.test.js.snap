// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComicsView should match the snapshot 1`] = `ShallowWrapper {}`;
